src/App.js

```diff
@@ -1,17 +1,21 @@
 import React from 'react';
+import { TodoList } from './TodoList';
 import './App.css';
 
 import users from './api/users';
+import todos from './api/todos';
+
+const preparedTodos = todos.map(todo => ({
+  ...todo,
+  user: users.find(user => user.id === todo.userId),
+}));
 
 function App() {
   return (
     <div className="App">
       <h1>Add todo form</h1>
 
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
+      <TodoList users={users} todos={preparedTodos} />
```

Comment:
`TodoList` should only render list of todos from props.
All forms and manipulations with the list should be handled in the `App` component.